nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/servlets/ListenHTTPServlet.java

```diff
@@ -424,6 +424,8 @@ protected void proceedFlow(final HttpServletRequest request, final HttpServletRe
                         response.setStatus(this.returnCode);
                         asyncContext.complete();
                     }, t -> {
+                        logger.error("Failed to commit session. Returning error response to Remote Host: [{}] Port [{}] SubjectDN [{}] IssuerDN [{}]",
+                                new Object[] {request.getRemoteHost(), request.getRemotePort(), foundSubject, foundIssuer}, t);
```

Comment:
Thanks for adding the log message! The wrapping `Object[]` can be removed:
```suggestion
                                request.getRemoteHost(), request.getRemotePort(), foundSubject, foundIssuer, t);
```

Reply:
This object wrapping is very common due to legacy usage. Do you think we could deprecate `ComponentLog::void error(String msg, Object[] os, Throwable t)` ?

Reply:
@exceptionfactory I applied your suggestion. Also for the other occurrences in this class.